docs/src/main/asciidoc/security-identity-providers.adoc

```diff
@@ -9,18 +8,18 @@ include::_attributes.adoc[]
 :diataxis-type: concept
 :categories: security
 
-In the Quarkus Security framework, identity providers play a key role in authentication and authorization, providing services for storing and verifying user identities.
-The Jakarta Persistence `IdentityProvider` creates a `SecurityIdentity` instance used during user authentication to verify and authorize access requests, making your Quarkus application secure.
+In the Quarkus Security framework, identity providers play a crucial role in authentication and authorization, providing services for storing and verifying user identities.
+The Jakarta Persistence `IdentityProvider` creates a `SecurityIdentity` instance, which gets used during user authentication to verify and authorize access requests, making your Quarkus application more secure.
```

Comment:
I know this PR is unrelated to it :-), but these 2 sentences need to be updated as they are not quite correct, let me try

Reply:
Thanks for your suggestion below :point_down: